op-deployer/pkg/deployer/forge/binary.go

```diff
@@ -252,11 +269,11 @@ func getForgeVersion(ctx context.Context, forgePath string) (string, error) {
 	if err != nil {
 		return "", fmt.Errorf("exec %s --version failed: %w", forgePath, err)
 	}
-	// Example output: "forge Version: 1.3.1-v1.3.1" -> capture "v1.3.1"
-	re := regexp.MustCompile(`(?mi)^\s*forge\s+version:\s+\d+\.\d+\.\d+-\s*(v\d+\.\d+\.\d+)\s*$`)
+	// Example output: "forge Version: 1.3.1-v1.3.1" -> capture initial "1.3.1"
+	re := regexp.MustCompile(`(\d+\.\d+\.\d+)`)
```

Comment:
Alpine forge binaries are currently returning the following output from `forge --version` so I had to change the regexp to be more lenient:
```
"forge Version: 1.3.1-dev"
```